pkg/cmd/kafka/create/create.go

```diff
@@ -474,31 +461,6 @@ func promptKafkaPayload(opts *options, userQuotaType accountmgmtutil.QuotaSpec)
 		}
 	}
 
-	marketplaces, err := accountmgmtutil.GetValidMarketplaces(f.Context, f.Connection)
```

Comment:
Why removing this?

Reply:
If we want to ask user to pick marketplace details this should happen before size and feed back into size (this is current bug in relased version of the CLI). I do not see how interactive mode would work with marketplace right now. Would we fail/still require users to use flags for non interactive selects?